pkg/deployments/src/task.ts

```diff
@@ -91,19 +91,19 @@ export default class Task {
       return await this.check(name, args, libs);
```

Comment:
This check on `TaskMode.CHECK` is redundant as we do another one right inside `deploy`. We *could* remove it if we're confident that we're not going to have any logic happening before the deploy in `deployAndVerify` (which seems reasonable).

Reply:
Yes, but that'd cause for CHECK executions to attempt verification, which we don't want to.

Reply:
Ah true, I forgot how functions work for a bit.